src/main/java/org/cryptomator/ui/controls/FontAwesome5Spinner.java

```diff
@@ -0,0 +1,84 @@
+package org.cryptomator.ui.controls;
+
+import com.tobiasdiez.easybind.EasyBind;
+
+import javafx.animation.AnimationTimer;
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.scene.control.ProgressIndicator;
+import java.time.Duration;
+
+/**
+ * A progress indicator in the shape of {@link FontAwesome5Icon#SPINNER}. The single spinner segements are defined in the css in the `progress-indicator` class.
```

Comment:
why css instead of the existing glyph?

Reply:
Experiments with just rotating the glyph resulted in a "wobbly" rotation. My guess for this behaviour of it is that the bounding box of a glyph is not quadratic or properly computed.

Reply:
I assume what you see there is caused by not setting `rotate.setInterpolator(Interpolator.LINEAR);`

Reply:
It reduces the wobbling, but it is still present (-;